components/cipher/stream-cipher/src/stream_cipher.rs

```diff
@@ -122,11 +126,15 @@ where
     }
 
     /// Applies the keystream to the provided input text.
+    #[cfg_attr(
+        feature = "tracing",
+        tracing::instrument(level = "trace", skip(self), err)
```

Comment:
`InputTextConfig` has an [opaque debug implementation](https://github.com/tlsnotary/tlsn/blob/282309c8e18f77a45bbbe2100101810ce0c8b9c3/components/cipher/stream-cipher/src/config.rs#L69-L85). So this should not be a problem because `text` field is not shown.